cartridge-cli.lua

```diff
@@ -3833,7 +3906,7 @@ function Process:start_with_decorated_output()
     local ok, err = utils.write_file(self.pid_file, pid, tonumber('644', 8))
     if not ok then return nil, err end
 
-    fiber.create(log_pipes_forwarder, pipes, self.instance_name)
+    fiber.create(log_pipes_forwarder, pipes, self.fullname)
```

Comment:
`fullname` looks strange.

`instance_name`/`app_name`/...

Reply:
Fullname is `<instance_name>.<app_name>`. It is called `instance_id` on arguments parsing, I can use this name.

Reply:
`instance_id ` is better